spec/lib/blacklight/solr/search_builder_spec.rb

```diff
@@ -62,10 +61,6 @@
         expect(subject[:qt]).to eq blacklight_config[:default_solr_params][:qt]
       end
 
-      it "should take rows from search field definition where specified" do
```

Comment:
@jkeck confirmed that this behavior doesn't actually seem to work and is extremely confusing in any case.